src/swdmt/redistricting/LocationTest.java

```diff
@@ -1,11 +1,10 @@
 package swdmt.redistricting;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.number.OrderingComparison.greaterThan;
 import static org.hamcrest.number.OrderingComparison.lessThan;
-import org.junit.Test;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.Assertions;
```

Comment:
Since there are only two kinds of assertions associated with this import, us a static import for each, such as
```
import static org.junit.jupiter.api.Assertions.assertTrue;
import static org.junit.jupiter.api.Assertions.assertFalse;
```
Then revert all of the instances of method invocations from `Assertions.assertTrue` to `assertTrue` and from `Assertions.assertFalse` to `assertFalse`.